packages/components/src/Typography/css/Underline.css

```diff
@@ -0,0 +1,19 @@
+.solid {
+  text-decoration: underline solid var(--color-base-grey--300);
+  text-underline-offset: 4px;
```

Comment:
Is there a variable we should be using for this? @jlelford

Reply:
Are both of the requests here for the underline? If so the comment below would apply, if there is a variation I'm not understanding let me know.

Reply:
This is for the distance between the text and the underline (wondering what the 4px should be ).

Reply:
Oh gotcha, the distance looks good in the images you posted above. Do you think that should be the Atlantis 
`--space-smaller` variable rather than specific pixel size?